torchgeo/datasets/oscd.py

```diff
@@ -170,8 +200,8 @@ def get_image_paths(ind: int) -> list[str]:
                 )
 
             images1, images2 = get_image_paths(1), get_image_paths(2)
-            if self.bands == "rgb":
-                images1, images2 = images1[1:4][::-1], images2[1:4][::-1]
```

Comment:
Not sure why `[::-1]` was performed here

Reply:
Probably to convert BGR to RGB